src/js/modules/template.js

```diff
@@ -93,7 +93,14 @@ export default {
     if (match.indexOf('/embed/') < 0) {
       id.splice(1, 0, 'embed')
     }
-    return `<div class="ejs-embed ejs-ustream"><iframe src="//www.${id.join('/')}" height="${options.videoHeight}" width="${options.videoWidth}"></iframe></div>`
+	console.log("ustream");
+	console.log(id);
+    //check if '//www.' is there in the link
+    if(match.indexOf('/www\.ustream\.tv/') < 0){
+      return `<div class="ejs-embed ejs-ustream"><iframe src="//www.${id.join('/')}" height="${options.videoHeight}" width="${options.videoWidth}"></iframe></div>`
```

Comment:
prevent multiple returns instead assign the src to a variable and use that in the template.